<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShopManagerForm</class>
 <widget class="QWidget" name="ShopManagerForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="listVerticalLayout">
       <item>
        <widget class="QTreeWidget" name="treeWidget">
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <column>
          <property name="text">
           <string>ID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>ClientName</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>ProductName</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Price</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Date</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Amount</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Discount</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>TotalProce</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QToolBox" name="toolBox">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="inputPage">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>361</width>
         <height>323</height>
        </rect>
       </property>
       <attribute name="label">
        <string>&amp;Input</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <item>
         <widget class="QLabel" name="idLabel">
          <property name="text">
           <string>I&amp;D</string>
          </property>
          <property name="buddy">
           <cstring>idLineEdit</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="idLineEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <widget class="QLabel" name="clientIdLabel">
            <property name="text">
             <string>ClientID</string>
            </property>
            <property name="buddy">
             <cstring>clientIdcomboBox</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="clientIdcomboBox">
            <property name="enabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QLabel" name="productNameLabel">
              <property name="text">
               <string>ProductName</string>
              </property>
              <property name="buddy">
               <cstring>productNamecomboBox</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="productNamecomboBox"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item>
             <widget class="QLabel" name="priceLabel">
              <property name="text">
               <string>Price</string>
              </property>
              <property name="buddy">
               <cstring>priceLineEdit</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="priceLineEdit">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QSplitter" name="splitter_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QWidget" name="layoutWidget">
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QLabel" name="amountLabel">
              <property name="text">
               <string>&amp;Amount</string>
              </property>
              <property name="buddy">
               <cstring>amountLineEdit</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="amountLineEdit"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="layoutWidget">
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QLabel" name="discountLabel">
              <property name="text">
               <string>&amp;Discount</string>
              </property>
              <property name="buddy">
               <cstring>discountLineEdit</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="discountLineEdit"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="totalPriceLabel">
          <property name="font">
           <font>
            <family>휴먼둥근헤드라인</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>&amp;TotalPrice</string>
          </property>
          <property name="buddy">
           <cstring>totalPriceLineEdit</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="totalPriceLineEdit">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="font">
           <font>
            <family>휴먼둥근헤드라인</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="addHorizontalLayout">
          <item>
           <widget class="QPushButton" name="addPushButton">
            <property name="text">
             <string>&amp;Add</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="modifyPushButton">
            <property name="text">
             <string>&amp;Modify</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QSplitter" name="splitter_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QWidget" name="layoutWidget">
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="font">
               <font>
                <family>휴먼둥근헤드라인</family>
                <pointsize>10</pointsize>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>Client Info</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTreeWidget" name="clientTreeWidget">
              <property name="headerHidden">
               <bool>false</bool>
              </property>
              <attribute name="headerVisible">
               <bool>true</bool>
              </attribute>
              <column>
               <property name="text">
                <string>Name</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>PhoneNumber</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Address</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="layoutWidget">
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="font">
               <font>
                <family>휴먼둥근헤드라인</family>
                <pointsize>10</pointsize>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>Product Info</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTreeWidget" name="productTreeWidget">
              <column>
               <property name="text">
                <string>ProductName</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Price</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Count</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Country</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="searchPage">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>361</width>
         <height>323</height>
        </rect>
       </property>
       <attribute name="label">
        <string>&amp;Search</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QTreeWidget" name="searchTreeWidget">
          <property name="rootIsDecorated">
           <bool>false</bool>
          </property>
          <column>
           <property name="text">
            <string>ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>ClientName</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>ProductName</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Price</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Date</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Amount</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Discount</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>TotalPrice</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QComboBox" name="searchComboBox">
            <item>
             <property name="text">
              <string>ID</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>ClinetName</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>ProductName</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Price</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Date</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Amount</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Discount</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>TotalPrice</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="searchLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="searchPushButton">
          <property name="text">
           <string>S&amp;earch</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
